src/components/EntityStatus/EntityStatus.scss

```diff
@@ -47,6 +47,23 @@
         line-height: var(--yc-text-body-2-line-height);
 
         color: var(--yc-color-text-complementary);
+
+        &_size_l {
+            font-size: var(--yc-text-header-2-font-size);
+        }
+
+        &_state_blue {
+            color: var(--yc-color-infographics-info-heavy);
+        }
+        &_state_yellow {
+            color: var(--yc-color-infographics-warning-heavy);
+        }
+        &_state_orange {
+            color: var(--yc-color-base-warning-orange);
+        }
+        &_state_red {
+            color: var(--yc-color-infographics-danger-heavy);
+        }
```

Comment:
There is already classes with colors for status icons (`&__status-color { &_state_green {} }`. Can you somehow reuse them?